tasks.py

```diff
@@ -31,19 +32,21 @@ def test(
     well with explicit ``--opts="-m=xxx"`` - if ``-m`` is found in ``--opts``,
     ``--include-slow`` will be ignored!)
     """
-    if verbose and '--verbose' not in opts and '-v' not in opts:
+    if verbose and "--verbose" not in opts and "-v" not in opts:
```

Comment:
You should probably avoid unrelated style changes in pull requests. If general style changes are desired they should probably be a separate independent pull request.

Reply:
True that, but the line length with the new task tripped flake8, and instead of mucking around formatting it correctly I elected to just blacken the file and have done. I *can* remove this if you want, but I'm not convinced it's worth the extra CPU cycles on Travis 😄